src/clr-addons/collapse-expand-section/directives/collapse-expand-section-subtitle.directive.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Directive } from '@angular/core';
+
+@Directive({ selector: 'clr-ces-subtitle' })
```

Comment:
Please replace this directives with attributes (in using component) and attribute selector (in hosting component)